src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,19 +41,26 @@ export const TodoFilter: React.FC = () => {
           type="text"
           className="input"
           placeholder="Search..."
+          value={query}
+          onChange={event =>
+            setQuery(event.target.value.toLowerCase().trimStart())
```

Comment:
This logic should be used only when searching, the user should be able to enter capital letters
```suggestion
            setQuery(event.target.value)
```